deps/wrapper.cc

```diff
@@ -533,12 +511,20 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod)
         .method("Connect", &ray::gcs::GlobalStateAccessor::Connect)
         .method("Disconnect", &ray::gcs::GlobalStateAccessor::Disconnect);
 
-    mod.method("report_error", &report_error);
+    // https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/core_worker/core_worker.h#L284
+    mod.add_type<ray::core::CoreWorker>("CoreWorker")
+        .method("GetCurrentJobId", &ray::core::CoreWorker::GetCurrentJobId)
+        .method("GetCurrentTaskId", &ray::core::CoreWorker::GetCurrentTaskId)
+        .method("put", &put)
+        .method("get", &get)
+        .method("submit_task", &submit_task)
```

Comment:
it's not clear to me why we are exposing these are methods of this class when they're not defined that way.

Reply:
I added these as methods of this class only to avoid headaches with moving these definitions around. The way we've set these up in Julia these methods are associated with this type so doing this just indicates the relationship here. Definitely optional